Pass OCP_Defaults instance to LoginController constructor for NX13

panovotn opened the pull request:
#8 Fixed

Just passing OCP_Defaults to LoginController constructor as now required in Nextcloud 13.

Commit:
Pass OCP_Defaults instance to LoginController constructor for NX13

moonwolf-github commented:
Any plan to merge this? I'm tryiing to do SSO in my project and SAML seems an overkill to me.

lemmon committed:
v14

lemmon committed:
fixes

tomtomas99911 committed:
Update README.md

Info about requirement of editing nextcloud core file to get this plugin working.

panovotn committed:
Merge pull request #2 from lemmon/master

v14

panovotn committed:
Merge pull request #3 from tomtomas99911/patch-1

Update README.md
